ecosystem/lunar_strategy/data.md

```diff
@@ -0,0 +1,27 @@
+--- 
+Name: "Lunar Strategy", 
+Website: "https://www.lunarstrategy.com/", 
+Twitter: "https://twitter.com/lunarstrategy",
+Architecture: "Partners",
+Segment: "Marketing Agency",
+Category: "",
+Application-Category: "",
+Tags: "Marketing Agency",
```

Comment:
Hi @piotr-dziubecki , can we change the tag from marketing agency to "Partner" please, which is one of the ecosystem approved tags.